README.md

```diff
@@ -19,6 +19,86 @@ string "AAI" and "Local" (any case) in the name.
 
 See `oidc-provider-dynamodb-adapter.ts` for the PK and GSI definitions.
 
+#### Create local dynamo db table using Docker
+
+Make sure you have Docker installed, and run:
+
+```shell
+docker run --name aai-dynamodb -p 8000:8000 amazon/dynamodb-local
+```
+
+Confirm that it is running:
+```
+docker ps
+```
+
+Set up your AWS cli, then create a table by running: 
+
+```shell
+aws dynamodb create-table \
+   --table-name agha-aai-test-bed-local-dev \
+   --attribute-definitions AttributeName=modelId,AttributeType=S \
+   --key-schema AttributeName=modelId,KeyType=HASH \
+   --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 \
+   --endpoint-url http://localhost:8000
+```
+
+Set up the GSI's by running:
+
+```shell
+aws dynamodb update-table \
+--table-name agha-aai-test-bed-local-dev \
+--attribute-definitions AttributeName=modelId,AttributeType=S \
+--global-secondary-index-updates \
+ '{"Create":{"IndexName":"uidIndex","KeySchema":[{"AttributeName":"modelId","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":10,"WriteCapacityUnits":5},"Projection":{"ProjectionType":"ALL"}}}' --endpoint-url http://localhost:8000
```

Comment:
You possibly don't need to set the provisioned throughput - and it might make these instructions simpler.
Especially for local dynamo where I don't think the throughput is probably a concept.
But if its a mandatory item then that's fine - leave it in

Reply:
Makes sense! Unfortunately it is mandatory.